docs/ember-intl/app/snippets/docs/helpers/format-date-range/example-4/template.hbs

```diff
@@ -2,7 +2,7 @@
 {{format-date-range
   this.yesterday
   this.today
-  dateStyle="full"
-  timeZone="America/New_York"
+  day="numeric"
+  month="short"
```

Comment:
I went with an example that may be more realistic. Based on the `fmt2` example in https://github.com/ember-intl/ember-intl/pull/1931#issuecomment-2559669950.